defaults/main.yml

```diff
@@ -71,3 +71,5 @@ omero_ms_image_region_max_active_channels: 10
 nginx_cache_expiration_time: 60m
 nginx_cache_inactive_time: 120m
 nginx_cache_max_size: 25m
+
+nginx_cache_ms_cash_path: '/var/cache/nginx/ms'
```

Comment:
For the root directory, you used `/opt/omero/microservice`. Any reason not to go for `/var/cache/nginx/microservice` (or `/var/cache/nginx/omero-ms-image-region`) for consistency?